tasklist/client/src/v2/ProcessesTab/index.tsx

```diff
@@ -512,55 +275,18 @@ const ProcessesTab: React.FC = observer(() => {
                       ))}
                 </Grid>
               )}
+              <Pagination
+                totalPages={Math.ceil(totalItems / PAGE_SIZE)}
+                currentPage={paginationParams.currentPage}
+                onNextPage={handleNextPage}
+                onPreviousPage={handlePreviousPage}
+                className={styles.pagination}
+              />
```

Comment:
I noticed that this would still be visible while loading. It might be a good idea to either hide, show loading state or disable the buttons while the content is loading.

Reply:
Good catch, I fixed it now